### PR TITLE
added expandenv flag

### PR DESCRIPTION
**Pull request by maia-iyer**
Adding expandEnv flag to expand environment variables in Tornjak config

- [x] Concerns about expandEnv for the SPIRE config - the Tornjak container should not have access to the relevant environment variables
- [x] Need to update usage documentation

Signed-off-by: Maia Iyer <maia.raj.iyer@gmail.com>

### CODE REVIEW

**Review by mamy-CS — APPROVED** [review]
lgtm, let's update the documentation for this as well.

File: run_backend.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@ Usage: run_server [-c <file>] [-t <file>]
 
 -c | -config <file>: SPIRE Config File
 -t | -tornjak-config <file>: Tornjak Config File
+-expandEnv: expand environment variables expressed in config files
```
Comment (mrsabath) [inline run_backend.sh:10]: Is this a key=value pair? Can you provide a sample example here? Does it take a single or multiple env. variable. If multiple, what is the format? Thank you!

**Review by mrsabath — CHANGES_REQUESTED** [review]
Looks good, but a small nit in the documentation

File: run_backend.sh (hunk 1 above)
Comment (maia-iyer) [reply run_backend.sh]: i have edited the usage documenation in the `docs/config-tornjak-agent.md` file. Maybe this should be a different file name?

File: docs/config-tornjak-agent.md [hunk 2]
```diff
@@ -89,9 +90,9 @@ Runs the tornjak http server.
 | `-listen-addr` | listening address for tornjak agent               | :10000  | false    |
 | `-cert`        | CA Cert path for TLS                              |         | false    |
 | `-key`         | Key path for TLS                                  |         | false    |
-| `mtls-ca`      | CA path for mTLS CA                               |         | false    |
-| `tls`          | Enable TLS for http server                        | false   | false    |
-| `mtls`         | enable mTLS for http server (overwrites tls flag) | false   | false    |
+| `-mtls-ca`     | CA path for mTLS CA                               |         | false    |
```
Comment (mrsabath) [inline docs/config-tornjak-agent.md:93]: Nice catch! Thank you

**Review by mrsabath — APPROVED** [review]
LGTM

**Pull request merged**